adhocracy-plus/assets/scss/components/_tile.scss

```diff
@@ -181,9 +185,9 @@
         }
     }
 
-    .tile__image {
-        min-height: 20rem;
-    }
+    // .tile__image {
```

Comment:
do we still need this?

Reply:
don't know, my database is pretty empty so I don't think I tested all the tile types so wanted to test on dev

Reply:
no dead code though, so maybe remove and if turns out it's needed re-add it?

Reply:
I don't normally do it like that as it's a reminder, if the styling is off and we forgot that we deleted it they we might add additional needless code when all we had to do was uncomment